pybamm/models/submodels/external_circuit/experiment_events.py

```diff
@@ -0,0 +1,35 @@
+#
+# Model to impose the events for experiments
+#
+import pybamm
+
+
+class ExperimentEvents(pybamm.BaseSubModel):
+    """Model to impose the events for experiments."""
+
+    def __init__(self, param):
+        super().__init__(param)
+
+    def set_events(self, variables):
+        # add current and voltage events to the model
+        # current events both negative and positive to catch specification
+        n_cells = pybamm.electrical_parameters.n_cells
+        self.events.extend(
+            [
+                pybamm.Event(
+                    "Current cut-off (positive) [A] [experiment]",
+                    variables["Current [A]"]
+                    - abs(pybamm.InputParameter("Current cut-off [A]")),
```

Comment:
Does the current cut-off need to be added to the parameter values or does it get a default value if unspecified?

Reply:
It's an input parameter so needs to be provided by the user when solving. But the experiment class does this automatically.